app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= render partial: 'admin/header', locals: { name: "Application" } %>
+
+<h1><%= @application.applicant_name %></h1>
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Description: <%= @application.description %></h3>
+<h3>Pets on this Application:</h3>
+<table>
+  <thead>
+    <tr>
+      <th>Pet Name</th>
+      <th>Approve</th>
+      <th>Reject</th>
+      <th>Approval Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @application.pet_applications.each do |pet_application| %>
+      <tr>
+        <td><%= pet_application.pet.name %></td>
+        <td>
+          <% if pet_application.approval == nil && pet_application.pet.adoptable != false %> 
+            <%= form_tag admin_application_path(@application, id: pet_application.id), method: :patch do %>
+            <%= submit_tag 'Approve' %>
+            <% end %>
+          <% end %>
+        </td>
+        <td>
+          <% if pet_application.approval == nil %>
+            <%= form_tag admin_application_path(@application, id: pet_application.id), method: :patch do %>
+            <%= submit_tag 'Reject' %>
+            <% end %>
+          <% end %>
+        </td>
+        <td>
+          <% if pet_application.approval == true || pet_application.pet.adoptable == false %>
```

Comment:
When you have a complex condition and it decides whether to render just simple _text_ however, I think you could move this conditional into a model method. It is presentation logic, but for something simple like this, I think it's not an MVC violation to put this in the model so that a PetApplication method can be called to determine what text is displayed here. Then, you could just render `pet_application.status_text` to get the label that's necessary